.config.yaml

```diff
@@ -0,0 +1,42 @@
+# tbpore external tools parameters config file
+# DO NOT EDIT - parameters that should be changed by users are listed in the program options
+# these are parameters that are passive to be changed, so they are listed here
+
+mykrobe:
+  predict:
+    params: "-e 0.08 --ploidy haploid --force --format json --min_proportion_expected_depth 0.20 --species tb -m 8192MB"
```

Comment:
BTW the `--force` flag will override the skeletons, so maybe you can remove that if we are caching them now.
Also, memory of 2GB is more than sufficient

Reply:
Done in https://github.com/mbhall88/tbpore/pull/8/commits/f0bdc91b809bc2805ec50758dd7c7d8ddeedb04f